manifests/firewalld/rule.pp

```diff
@@ -0,0 +1,239 @@
+# Add firewall rules via firewalld
```

Comment:
Are you assuming this is fully tested in the spec/defines/listen/firewalld/*_spec.rb tests and the acceptance tests?  Didn't see an explicit unit test for it.

Reply:
It should be in now with the latest commits.